testfiles/CMakeLists.txt

```diff
@@ -483,35 +483,35 @@ endif ()
-    ADD_SIMULATION_TEST(IDF_FILE HAMT_HourlyProfileReport.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneBaseboardHeat.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneConstantVolumeChillerBoiler.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneDualDuct.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneFanCoil-DOAS.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneFanCoil.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneFurnaceDX.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZonePTAC-DOAS.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZonePTAC.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZonePTHP.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZonePackagedVAV.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZonePurchAir.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneUnitaryHeatPump.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneUnitarySystem.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneVAVFanPowered.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneVAVWaterCooled-ObjectReference.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneVAVWaterCooled.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneVRF.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE HVACTemplate-5ZoneWaterToAirHeatPumpTowerBoiler.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE LBuilding-G000.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE LBuilding-G090.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE LBuilding-G180.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
-    ADD_SIMULATION_TEST(IDF_FILE LBuilding-G270.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+    ADD_SIMULATION_TEST(IDF_FILE HAMT_DailyProfileReport.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw COST 2)
```

Comment:
If basement starts at the end of the test runs, everything else will be done and it will still churn for like 20 minutes.  If basement starts first, at least the CPU will be pegged more of the time and not waiting on a single thread to finish.